src/components/Slider/Slider.tsx

```diff
@@ -54,18 +56,31 @@ class Slider extends Component<SliderProps> {
   }
 
   // creates an array of values where the slider should snap to
-  private computeSnapPoints({ min, max, step }) {
+  private computeSnapPoints({
+    min,
+    max,
+    step,
+  }: {
+    min: number;
+    max: number;
+    step?: number;
+  }) {
     if (!step) return undefined;
     return [...range({ start: min, end: max, step }), max];
   }
 
-  private createHandleComponent = tooltips => props => {
+  private createHandleComponent = (
+    tooltips: RangeSliderWidgetParams['tooltips']
+  ) => (props: HandleProps) => {
     // display only two decimals after comma,
     // and apply `tooltips.format()` if any
     const roundedValue =
-      Math.round(parseFloat(props['aria-valuenow']) * 100) / 100;
+      Math.round(
+        // have to cast as a string, as the value given to the prop is a number, but becomes a string when read
+        parseFloat((props['aria-valuenow'] as unknown) as string) * 100
```

Comment:
isn't `'aria-valuenow'` optional? I think casting as unknown hide the fact this value can be undefined but I'm not yet sure how we can avoid that.

Reply:
```suggestion
        parseFloat((props['aria-valuenow'] as unknown) as (string | undefined)) * 100
```

is this better?

Reply:
parseFloat doesn't accept undefined, so it will be for some other time when we figure out why it even needs to be parsed as a float if it's already a number